### DISCUSSION TITLE
Why Per-component translations not working?

### DISCUSSION THREAD

**Discussion by mrFANRA** [nuxt-modules/i18n#2127] [Q&A] [answered]
Hi!

Why Per-component translations not working?

Reproduction:
https://stackblitz.com/edit/nuxt-starter-o8kovb?file=components%2Fdemo-component.vue

**Comment by steinerd** [comment, marked as answer]
https://stackblitz.com/edit/nuxt-starter-zczwyp?file=components%2Fdemo-component.vue

I forked it. 

You'll need to add, and replace `$t()` with just `t()`

```
const { t } = useI18n({ useScope: 'local' });
```
Ideally in the setup routine. 

And you'll optionally want to add the `scoped` attribute to your `<i18n>` custom block.

**Reply by mrFANRA** [reply]
Damn... it is not documented. Thanks!

**Reply by steinerd** [reply]
It's documented, it's just not in the most obvious place. 
This module is a superset of the vue-i18n module; I found the answers on https://vue-i18n.intlify.dev/guide/advanced/sfc.html
